docs/useful_tools_scripts.md

```diff
@@ -111,6 +111,17 @@ python tools/publish_model.py work_dirs/dff_faster_rcnn_r101_dc5_1x_imagenetvid/
 
 The final output filename will be `dff_faster_rcnn_r101_dc5_1x_imagenetvid_20201230-{hash id}.pth`.
 
+### SiameseRPN++ Test-time Parameter Search
```

Comment:
Put it under `## MOT Test-time Parameter Search` section

remeber to modify the chinese docs